### ISSUE TITLE
SnackProgressBar motion animation of attached views

### ISSUE THREAD

**Issue by 0x7e57** [tingyik90/snackprogressbar#32]
When the root view is not a CoordinatorLayout the animation of attached views is not synchronized with the SnackProgressBarLayout animation.
`                .setViewsToMove(new View[]{
                        viewBinding.fab,
                })`
how to remove animation delay of attached views?

**Comment by tingyik90** [comment]
Thanks for the good catch. This was due to the `BaseTransientBottomBar` using `FastOutSlowInInterpolator` which was not applied to `viewsToMove`. See commit 922c1eb5bb869e22ab6e0b4f226566235a8d3b23. Try `implementation 'com.github.tingyik90:snackprogressbar:master-SNAPSHOT'` and let me know if this solves the animation issue. 

I've tested it on an emulator and slowed the animation duration by 10x and it matches.
